<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Extração de dados de web sites</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="resultDisplay">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>511</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>550</y>
      <width>481</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Pronto</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>511</width>
      <height>111</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="urlInput">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="placeholderText">
        <string>Cole a URL aqui</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QComboBox" name="dataSelector">
          <item>
           <property name="text">
            <string>Títulos</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Paragrafos</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Links</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Imagens</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Cabeçalhos</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Listas</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Tabelas</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Metadados</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Scripts</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="executeButton">
          <property name="text">
           <string>Executar</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
